chapters/license-matching-guidelines-and-templates.md

```diff
@@ -189,16 +237,7 @@ Rule fields:
 
 The [POSIX ERE]( http://pubs.opengroup.org/onlinepubs/9699919799/) in the match field has the following restrictions and extensions:
 
-```text
-Semicolons are escaped with \;
-```
-
-```text
-POSIX Bracket Extensions are not allowed
-```
-
-EXAMPLE:
+* Semicolons are escaped with \;
```

Comment:
You may need to put backticks around `\;`, otherwise the backslash doesn't show up in the rendered MarkDown. They weren't needed in the prior version because it was previously enclosed in the `text` lines.

Reply:
yeah, I thought it was weird that the instructional bullets (or that's how I read them) were in shown as "code"

Reply:
I think I've fixed this in the next commit, but please check...

Reply:
Yup, looks good to me!